app/facades/parks_facade.rb

```diff
@@ -0,0 +1,13 @@
+class ParksFacade
+  def self.create_park(state)
+    json = ParksService.get_parks_data(state)
+    json[:data].map do |data|
+      Park.new(data)
+    end
+  end
+
+  def self.total_parks_found(state)
```

Comment:
If you wanted to provide both the parks and the total parks found, you could do this with one request! Consider the idea of returning a collection from your facade method rather than just the parks or just the total. You could extract the parks and the total in the same method, and then return `[parks, total]` back to your controller so that you're able to display both easily in the view.